qt/widgets/instrumentview/src/InstrumentActor.cpp

```diff
@@ -1051,13 +1051,16 @@ void InstrumentActor::setDataIntegrationRange(const double &xmin,
     m_DataMaxValue = -DBL_MAX;
 
     const auto &spectrumInfo = workspace->spectrumInfo();
+    auto maskWksp = getMaskWorkspace();
```

Comment:
Can I just check that `maskWsp` can never be null here?

Reply:
`getMaskWorkspace` initializes a mask workspace if there was none, so I don't think it can return null. Do you prefer to double-check nonetheless ?

Reply:
No not at all. I hadn't appreciated that there would be a mask workspace even if no mask was applied so I just wanted to check. Thanks.